PROTO_tests/tests/array_view_test.py

```diff
@@ -0,0 +1,201 @@
+import random
+from itertools import product
+
+import numpy as np
+import pytest
+
+import arkouda as ak
+
+SHAPE = {
+    4: (2, 2),
+    30: (5, 3, 2),
+    252: (4, 9, 7),
+    693: (3, 7, 3, 11),
```

Comment:
I think we should lower these numbers especially for the ones that have more dimensions. For `test_reshape_order` we do a distinct index for each combination and that can take a while with multi-locale. 
so I would shift the larger dimension sizes to the ones with few dimensions

Reply:
Updated